recipes/protobuf/all/patches/upstream-pr-4709-remove-js_embed.patch

```diff
@@ -0,0 +1,470 @@
+diff --git a/.gitignore b/.gitignore
+index af88e2260..b07e9f5d6 100644
+--- a/.gitignore
++++ b/.gitignore
+@@ -47,7 +47,6 @@ any_test.pb.*
+ map*unittest.pb.*
+ unittest*.pb.*
+ cpp_test*.pb.*
+-src/google/protobuf/compiler/js/well_known_types_embed.cc
+ src/google/protobuf/util/**/*.pb.cc
+ src/google/protobuf/util/**/*.pb.h
+ 
+diff --git a/cmake/libprotoc.cmake b/cmake/libprotoc.cmake
+index 107c1c526..e462bd8e8 100644
+--- a/cmake/libprotoc.cmake
++++ b/cmake/libprotoc.cmake
+@@ -203,18 +203,6 @@ set(libprotoc_headers
+   ${protobuf_source_dir}/src/google/protobuf/compiler/plugin.pb.h
+ )
+ 
+-set(js_well_known_types_sources
+-  ${protobuf_source_dir}/src/google/protobuf/compiler/js/well_known_types/any.js
+-  ${protobuf_source_dir}/src/google/protobuf/compiler/js/well_known_types/struct.js
+-  ${protobuf_source_dir}/src/google/protobuf/compiler/js/well_known_types/timestamp.js
+-)
+-add_executable(js_embed ${protobuf_source_dir}/src/google/protobuf/compiler/js/embed.cc)
+-add_custom_command(
+-  OUTPUT ${protobuf_source_dir}/src/google/protobuf/compiler/js/well_known_types_embed.cc
+-  DEPENDS js_embed ${js_well_known_types_sources}
+-  COMMAND js_embed ${js_well_known_types_sources} > ${protobuf_source_dir}/src/google/protobuf/compiler/js/well_known_types_embed.cc
+-)
+-
+ add_library(libprotoc ${protobuf_SHARED_OR_STATIC}
+   ${libprotoc_files} ${libprotoc_headers})
+ target_link_libraries(libprotoc libprotobuf)
+diff --git a/src/google/protobuf/compiler/js/embed.cc b/src/google/protobuf/compiler/js/embed.cc
+deleted file mode 100644
+index a725b62eb..000000000
+--- a/src/google/protobuf/compiler/js/embed.cc
++++ /dev/null
```

Comment:
Why don't you keep this file if it is not used anymore? Patch would be simplier.